GiniVision/Classes/Core/Logger.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  Logger.swift
+//  GiniVision
+//
+//  Created by Enrique del Pozo Gómez on 5/14/18.
+//
+
+import Foundation
+
+final class Logger {
+    
+    enum Event {
+        case error
+        case success
+        case warning
+        case custom(emoji: String)
+        
+        var value: String {
+            switch self {
+            case .error: return "❌"
```

Comment:
Have you tried extracting the logs from a real device (from iTunes or Xcode)? Can you verify that these emojis show? It's a real use case that someone gets logs from a device (from a crash reporter for instance) and tries to make sense of them.

Reply:
They are shown if I use `NSLog` (the whole message, including emojis).